packages/docs/docs/sidekicks-quickstart.md

```diff
@@ -27,212 +34,193 @@ At the end of this quickstart you will:
+> TODO: Add more information about the kinds of queries you can ask of the template
+> Sidekick based on whatever the public corpus ends up being.
 
-From your terminal:
+## Local Development and Testing
 
-```terminal
-npx @fixieai/fixie deploy
-```
+The `fixie deploy` step can take a couple of minutes to build and deploy your
+Sidekick to the cloud, which is a real bummer when you're testing things locally.
+Fortunately, you can run your Sidekick locally, without needing to deploy it to
+the cloud.
 
-Deploying takes up to a couple of minutes. While deployment is running you will see status messages in your terminal. For example, if our Fixie user is "sarah" and she is deploying a sidekick named "sidekick-acme", then we would expect to see something like this:
+Instead of `fixie deploy`, you run:
 
 ```terminal
-sarah@computer % FIXIE_API_URL='https://console.fixie.ai' npx @fixieai/fixie deploy
-🦊 Deploying agent sarah/sidekick-acme...
-👽 Updating agent sarah/sidekick-acme...
+$ fixie serve
 ```
 
-Once complete deployment completes, we would expect to see something like this:
+This starts up the Sidekick running on your local machine, and sets
+up a tunnel allowing the Fixie service to connect into your local
+Sidekick. When you quit the `fixie serve` command (for example, by
+hitting Ctrl-C), the Sidekick reverts back to the most recently
+deployed version (from `fixie deploy`). Note that you need to `fixie
+deploy` your Sidekick in order for it to run in the cloud.
 
-```terminal
-sarah@computer % FIXIE_API_URL='https://console.fixie.ai' npx @fixieai/fixie deploy
-🦊 Deploying agent sarah/sidekick-acme...
-👽 Updating agent sarah/sidekick-acme...
-✔ Revision a9b5e04c was deployed to https://console.fixie.ai/agents/sarah/sidekick-acme
-```
-
-#### Troubleshooting: Verify the Sidekick Builds
-
-If the above deploy step failed for some reason, it is a good idea to build our sidekick locally to see if there are any errors raised. Building is done from the terminal:
-
-```terminal
-npm build
-```
-
-_TODO Matt: Is this still needed?_
+As we make changes to our sidekick, we can simply stop serving our sidekick with `Control+C` and then serve up our new changes with the `serve` command as we did above.
 
-## Step 3: Converse with Sidekick
 
-<img src={Step3Profit} alt="Fixie profile page where you can get your API key." width="300"/>
+## Sending messages to your Sidekick
 
-We've got our Sidekick deployed. Let's test it out to see what it can do!
+The Fixie Console page gives you a simple web interface to interact with your
+Sidekick, but you're not limited to this interface. You can chat with your
+Sidekick directly via a REST API, or embed the Sidekick chat UI in your own
+web app.
 
-We are going to test the sidekick using four different methods.
+### Method 1: Via the REST API
 
-### Method 1: Via curl
+> TODO(matt): Figure out the right invocation for this.
```

Comment:
See https://github.com/fixie-ai/fixie/blob/5452e48b739f9941e3facabaad9a2f1d4f59f49c/src/generic-sidekick-frontend/lib/fixie-api.ts for an example of how this API is used.